src/Gaffer/Reference.cpp

```diff
@@ -172,6 +174,15 @@ void Reference::loadInternal( const std::string &fileName )
 				convertPersistentMetadata( it->get() );
 			}
 		}
+		else if( Node *node = runTimeCast<Node>( newChildren->member( i ) ) )
```

Comment:
If its negligible I don't mind. I didn't look into the details, I just saw a lookup into a set and assumed it would be slower, and Yuta had just made that other ticket about slow load times...